website/src/docs/strawberryshake/subscriptions.md

```diff
@@ -36,7 +36,7 @@ subscription OnSessionUpdated {
 builder.Services
   .AddConferenceClient()
   .ConfigureHttpClient(client => client.BaseAddress = new Uri("http://localhost:5050/graphql"))
-  .ConfigureWebSocketClient(client => client.Uri = new Uri("ws://localhost:5050/graphql"));
+  .ConfigureWebSocketClient(client => client.Uri = new Uri("wss://localhost:5050/graphql"));
```

Comment:
this was correct before since the example shows a non-secure web socket connection ... in this example we are using http and ws as opposed to https and was.